test/WebSites/BasicWebSite/Controllers/JsonResultController.cs

```diff
@@ -32,6 +33,14 @@ public JsonResult CustomContentType()
             return result;
         }
 
+        public JsonResult CustomSerializerSettings()
+        {
+            var serializerSettings = new JsonSerializerSettings();
+            serializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
```

Comment:
Cache this :laughing:. Since we recommend that users do it, it would be good of the test/sample code did it too.